orca-sql/src/main/resources/db/changelog/20240731-add-compression-type-column-mariadb.yaml

```
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-pipelines-compressed-executions-mariadb
```

```suggestion
      id: add-compression-type-column-to-pipelines-compressed-executions-mariadb
```
and similar changes below.